### PR TITLE
Content Additions

### PR DESCRIPTION
**Pull request by quarterback**
Meant to address #471, #611, and #497. 
Comments in each issue for devs. Adds content sections for submit/certify, as well as assurances text boxes which need to be replaced to be actual checkboxes. Added submit text copy. HIE activity section added to yaml, design of the section (a text box with specific text) mockup included inside of issues.

### CODE REVIEW

File: web/src/i18n/locales/en.yaml [hunk 1]
```diff
@@ -245,28 +256,28 @@
...
+      helpText: "If the APD is approved, these program budget tables will be referenced on the CMS approval letter."
     budgetTable: 
       title: "Program Budget Table"
       helpText: ""
   certifyAndSubmit: 
     title: "Certify and Submit"
-    helpText: "By clicking Submit, you agree that your funding request fulfills our standards and requirements."
-    certify: 
+    helpText: "Note, this application will be locked upon submission until after a CMS analyst reviews it."
+    certify: "Note, this application will be locked upon submission until after a CMS analyst reviews it."
```
Comment (mgwalker) [inline web/src/i18n/locales/en.yaml]: This `certify` is a mapping entry, meaning it has children.  An entry cannot have both a text value and children.  So this is invalid:

```yaml
certify: "some text"
    title: "other text"
    subtitle: "bloop"
```

but this is valid:

```yaml
certify:
    name: "some text"
    title: "other text"
    subtitle: "bloop"
```

Which one is meant here?  It looks like `certify` and `helpText` have the same text value, so I'm guessing only `helpText` needs that content?

Comment (quarterback) [reply web/src/i18n/locales/en.yaml]: Yeah, only help text needs the content.

File: web/src/i18n/locales/en.yaml [hunk 2]
```diff
@@ -75,6 +75,17 @@
...
+title: "HIE Activity Details"	
+helpText: "Describe the activity in detail including images, diagrams, or other relevant information. If this an HIE activity, the activity details must also include a description of three items: 	
+- How the activity is linked to meaningful use;	
+- Value proposition to providers;	
+- Sustainability plan describing how the activity will be sustained after HIE DDI funding is exhausted.	
+Additionally, include diagrams and descriptions of other HIE linkages as needed if they provide more context for how you'll complete the activity."
+list:
+title: "Activity List"	
+title: "Program Activities"
+helpText: "Identify each of the activities referenced from the overview section. CMS will collect information for each activity."
```
[the remark this answers is not in the document]
Comment (quarterback) [reply web/src/i18n/locales/en.yaml]: I can reconstruct that to make it easier. But yes, all of that belongs to HIE activities.

**Comment by quarterback** [comment]
Fixing the indentation and redundancies, merging with other changes & will resubmit.

**Pull request closed without merging**